File: nyatado/min_to_sec.cpp
#include<iostream>
using namespace std;
int results(int min);
int main(){
	int minutes,seconds;
	cout<<"Enter the number of minutes\n";
	cin>>minutes;
	
	seconds=results(minutes);
	cout<<"The number of seconds is :"<<seconds;
	return 0;
	
}
int results(int min)
{
	return(min*60);
}

File: Michael-solonka/min_to_sec.java
class min_to_sec
{
    public static int  convert(int x)
    {
        int z=0;
        z=x*60;
        return z;
    }
   public static void main(String[] args) {
       
   
        int z;
        z=convert(5);
        System.out.println("5 minutes are " +z+" seconds");

        z=convert(3);
        System.out.println("3 minutes are " +z+" seconds");

        z=convert(2);
        System.out.println("2 minutes are " +z+" seconds");
    
       
    }

}
